linera-execution/src/system.rs

```diff
@@ -858,13 +852,19 @@ where
                     Recipient::Burn => (),
                 }
             }
-            SetCommittees { epoch, committees } => {
+            CreateCommittee { epoch, committee } => {
                 ensure!(
-                    epoch >= self.epoch.get().expect("chain is active"),
-                    SystemExecutionError::CannotRewindEpoch
+                    epoch == self.epoch.get().expect("chain is active").try_add_one()?,
+                    SystemExecutionError::InvalidCommitteeCreation
                 );
+                self.committees.get_mut().insert(epoch, committee.clone());
                 self.epoch.set(Some(epoch));
-                self.committees.set(committees);
+            }
+            RemoveCommittee { epoch } => {
+                ensure!(
+                    self.committees.get_mut().remove(&epoch).is_some(),
+                    SystemExecutionError::InvalidCommitteeRemoval
+                );
             }
             Subscribe { id, subscription } => {
```

Comment:
We don't execute this code any more. Do we?

Reply:
I can see why it's complicated to make a similar PR for applications now: we would have to use this message for all applications, and change the system APIs to create this message instead of managing the list of subscribers.

Reply:
We might be forced to include an incoming bundle that contains a `Subscribe` or `Unsubscribe` message. But it should be a no-op now; I'll remove the code.